cvmfs/options.cc

```diff
@@ -74,10 +54,38 @@ static string EscapeShell(const std::string &raw) {
 }
 
 
-void ParsePath(const string &config_file, const bool external) {
-  LogCvmfs(kLogCvmfs, kLogDebug, "Parsing config file %s", config_file.c_str());
+void FastOptionsManager::ParsePath(const string &config_file, const bool external) {
```

Comment:
You can make the fact that the `external` parameter is unused explicit by `__attribute__((unused))`